api/conf/routes

```diff
@@ -1,4 +1,6 @@
-# Admin 
+GET     /healthcheck                       controllers.HealthcheckController.get
```

Comment:
Maybe move this to the bottom of the routes alongside the other public endpoint?